README.md

```diff
@@ -0,0 +1,45 @@
+# Galoy CLI
+Galoy CLI is a Rust-based CLI client that can interact with the Galoy backend using GraphQL queries and mutations.
+
```

Comment:
## Rust installation and getting started with cargo commands
view this guide https://www.rust-lang.org/learn/get-started